.travis.yml

```diff
@@ -7,11 +7,11 @@ go:
   - 1.9
 
 install:
-  - go get github.com/Masterminds/glide
+  - go get github.com/golang/dep/cmd/dep
```

Comment:
Can we use the binaries here instead? Otherwise we might use something unstable in the tests - plus it reduces build time.

Reply:
sure, will fix that